templates/open-webui/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3.8'
+services:
+
+  ollama:
+    volumes:
+      - ollama:/root/.ollama
+    container_name: ollama
+    networks:
+      - dokploy-network
+    pull_policy: always
+    tty: true
+    restart: unless-stopped
+    image: ollama/ollama:${OLLAMA_DOCKER_TAG-latest}
+
+  open-webui:
+    image: ghcr.io/open-webui/open-webui:${WEBUI_DOCKER_TAG-main}
```

Comment:
can we have a defined version, since this can be break in the future?

Reply:
I've added the version in ENV variables